wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/templates/xrpeducational/EducationalRobot.java

```diff
@@ -34,8 +34,6 @@ public void test() {
 
   @Override
   public void startCompetition() {
-    robotInit();
-
```

Comment:
Does this still make sense given that we aren't deprecating quite yet?

Reply:
>  What we should do first is update all the templates, examples, and documentation to not use/have it, and then we can deprecate it a year later.